test/controllers/graphql_controller_test.rb

```diff
@@ -105,11 +106,23 @@ def setup
 
   test "should read medias" do
     assert_graphql_read('media', 'url')
-    assert_graphql_read('media', 'jsondata')
     assert_graphql_read('media', 'published')
     assert_graphql_read('media', 'last_status')
   end
 
+  test "should read medias jsondata" do
+    authenticate_with_user
+    p = create_project team: @team
+    m = create_valid_media project_id: p.id
+    query = "query GetById { media(id: \"#{m.id}\") { jsondata(context: \"#{p}\") } }"
+    post :create, query: query
+    assert_response :success
+    # calling without context
+    query = "query GetById { media(id: \"#{m.id}\") { jsondata(context: nil) } }"
+    post :create, query: query
+    assert_response :success
+  end
```

Comment:
I was expecting something different for this test. Two medias are created, for two different projects. One query for each project, then we assert that only the relevant media was returned. Then a third case for when a context is not provided. Also, the client doesn't send it as `nil`, but as an empty string.